src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt

```diff
@@ -297,6 +298,11 @@ class VisitService(
     log.info("Visit Id filter request : $visitFilter with page request $pageRequest")
     return visitRepository.findAll(VisitSpecification(visitFilter), pageRequest).map { VisitIdResponse(it.id) }
   }
+
+  fun findRoomCountsByFilter(visitFilter: VisitFilter): List<VisitRoomCountResponse> {
+    log.info("Visit room count request with filter: $visitFilter")
```

Comment:
not sure this log line is adding any value?  App insights will log the requests with their request parameters anyway so you should get this for free

Reply:
removed